tests/rbd/test_rbd_image_migration_qcow.py

```diff
@@ -92,13 +93,15 @@ def test_qcow_image_migration(rbd_obj, client, **kw):
                 action="execute",
                 dest_spec=image_spec,
                 client_node=client,
+                long_running=True,
                 **kw,
             )
 
             # verify execute migration status
             if verify_migration_state(
                 action="execute",
                 image_spec=image_spec,
+                long_running=True,
```

Comment:
@chaitanyadommeti add `long running` for all three actions such migration prepare, execute, commit